### PR TITLE
Reporting and analysis

### PR DESCRIPTION
**Pull request by Jamingzor127**

### CODE REVIEW

File: src/pages/reporting.tsx [hunk 1]
```diff
@@ -1,7 +1,128 @@
...
+  { name: 'Group D', value: 200 },
+  { name: 'Group E', value: 278 },
+  { name: 'Group F', value: 189 },
+];
+
+const colors: string[] = ["#FFBF00", "#DE3163", "#CCCCFF", "#6495ED", "#9FE2BF", "#40E0D0"];
+
 const ReportingPage: NextPageWithLayout = () => {
-  return <div>Reporting & Analysis</div>;
+  return (<div className="flex flex-row w-full py-24" style={{justifyContent: 'space-around', overflow: 'auto'}}>
```
Comment (zthh626) [inline src/pages/reporting.tsx]: was prettier run here?

just needs linting otherwise looks good

**Review by zthh626 — CHANGES_REQUESTED** [review]

File: src/pages/reporting.tsx (hunk 1 above)
Comment (Jamingzor127) [reply src/pages/reporting.tsx]: No I forgot to run linting lol, will do

Comment (Jamingzor127) [reply src/pages/reporting.tsx]: Updated

File: src/pages/reporting.tsx [hunk 2]
```diff
@@ -1,7 +1,188 @@
...
+  "#6495ED",
+  "#9FE2BF",
+  "#40E0D0",
+];
+
 const ReportingPage: NextPageWithLayout = () => {
-  return <div>Reporting & Analysis</div>;
+  return (
+    <div
+      className="flex flex-row w-full py-24"
```
Comment (zthh626) [inline src/pages/reporting.tsx]: https://tailwindcss.com/docs/justify-content

`space-around` and i believe overflow is also in there.

try and use tailwind wherever possible

File: src/pages/reporting.tsx [hunk 3]
```diff
@@ -1,7 +1,188 @@
...
+      style={{ justifyContent: "space-around", overflow: "auto" }}
+    >
+      <div
+        className="flex"
+        style={{
```
Comment (zthh626) [inline src/pages/reporting.tsx]: same with this

**Review by zthh626 — CHANGES_REQUESTED** [review]

**Review by zthh626 — APPROVED** [review]

**Pull request merged**